SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex

```diff
@@ -4273,6 +4273,22 @@ \subsection{Documenting Data Members}
 \code{Set\#\#name} method should also be documented following the above
 guidelines.
 
+Data members that should share the same documentation description should be embedded in
+the Doxygen grouping commands by means of the comments \code{/** @ITKStartGrouping */}
```

Comment:
by means of : using ?

Reply:
@jhlegarreta Not sure if this is meant as correction or as a style / discussion item from your side?

Reply:
Sounds more straightforward.